vars/util.groovy

```diff
@@ -424,7 +424,7 @@ boolean isJobResultUnstable(String jobResult) {
     return jobResult == 'UNSTABLE'
 }
 
-String getMarkdownTestSummary(String jobId, String buildUrl = "${BUILD_URL}") {
+String getMarkdownTestSummary(String jobId, String additionalInfo = '', String buildUrl = "${BUILD_URL}") {
```

Comment:
in case you change a method contract for an already used one adding an optional argument, it is good to add it at the end to keep backward compatibility

Reply:
yeah I know ... but I know for sure this is not yet used as I just created it yesterday and this will be used by https://github.com/kiegroup/drools/pull/4064 and related PRs

Reply:
Impact should be null